bot_cli/commands.py

```diff
@@ -264,3 +263,43 @@ def birthdays(args):
         print_success(result)
     else:
         print_warn(f"There is no one to celebrate birthday for next {period} day(s)")
+
+
+@contact_not_found_error
+@add_address_error
+def add_address(args):
+    try:
+        name, *address = args
```

Comment:
`addresses` -> `address_lines` ?

Reply:
There is 1 line (* - means truple of strings). Not sure it makes sense to change the variable because all the examples in the code uses same syntax.